Nuget/Ploeh.AutoFixture.AutoFakeItEasy2.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>AutoFixture.AutoFakeItEasy2</id>
+        <version>@build.number@</version>
+        <title>AutoFixture with Auto Mocking using FakeItEasy</title>
+        <authors>Christopher Loggins</authors>
+        <owners>Christopher Loggins</owners>
+        <dependencies>
+            <dependency id="AutoFixture" version="@build.number@" />
+            <dependency id="FakeItEasy" version="2.0" />
```

Comment:
Unless we already know that FakeItEasy 3 will have changes that break this library, I see no reason to limit the dependency to less than version 3.

As an example, [AutoFixture.AutoMoq](https://www.nuget.org/packages/AutoFixture.AutoMoq) for years depended on Moq 3.1, but it worked fine with Moq 4.

While there may have been breaking changes between Moq 3.1 and Moq 4, they didn't affect AutoFixture.AutoMoq.